src-opam/dockerfile_windows.ml

```diff
@@ -228,6 +231,9 @@ module Winget = struct
 
   module Git = struct
     let init ?(name="Docker") ?(email="docker@example.com") () =
-      run "git config --global user.email %S && git config --global user.name %S && git config --system core.longpaths true" email name
+      run "git config --global user.email %S && \
+           git config --global user.name %S && \
+           git config --global core.longpaths true && \
```

Comment:
Note that the user doesn't have the permissions to set directives system-wide, so I also fixed `core.longpaths` to `--global`. By default, cygwin's git is used.

Reply:
Is it possible to do this earlier in a step as ContainerAdministrator?

Reply:
I re-tested and must have misread, the `--system` switch for `core.longpaths` worked as expected, both for Cygwin's git and Git for Windows. Reverting that change.
As a side note, the user is always Container Administrator. I haven't created another user.